various-readers/src/main/java/com/dellemc/oe/readers/FlinkSQLReader.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package com.dellemc.oe.readers;
+
+import com.dellemc.oe.readers.util.EarthQuakeRecord;
+import com.dellemc.oe.util.CommonParams;
+import com.dellemc.oe.util.Constants;
+import io.pravega.client.stream.Stream;
+import io.pravega.connectors.flink.*;
+import org.apache.flink.streaming.api.TimeCharacteristic;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.java.StreamTableEnvironment;
+import org.apache.flink.table.api.Table;
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.descriptors.Json;
+import org.apache.flink.table.descriptors.StreamTableDescriptor;
+import org.apache.flink.table.factories.StreamTableSourceFactory;
+import org.apache.flink.table.factories.TableFactoryService;
+import org.apache.flink.table.sources.TableSource;
+import org.apache.flink.table.descriptors.Schema;
+import org.apache.flink.types.Row;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.util.Map;
+
+/*
+ *  This flink application demonstrates the JSON Data reading
+ */
+public class FlinkSQLReader {
+
+    // Logger initialization
+    private static final Logger LOG = LoggerFactory.getLogger(FlinkSQLReader.class);
+
+    public static void main(String[] args) throws Exception {
+        LOG.info("########## FlinkSQLReader START #############");
+        CommonParams.init(args);
+        final String scope = CommonParams.getParam(Constants.SCOPE);
+        String streamName = CommonParams.getParam(Constants.STREAM_NAME);
+        final URI controllerURI = URI.create(CommonParams.getParam(Constants.CONTROLLER_URI));
+
+        LOG.info("#######################     SCOPE   ###################### " + scope);
+        LOG.info("#######################     streamName   ###################### " + streamName);
+        LOG.info("#######################     controllerURI   ###################### " + controllerURI);
+        run(scope, streamName, controllerURI);
+    }
+
+    public static void run(String scope, String streamName, URI controllerURI) {
+
+        try {
+            // Read from jason stream
+            streamName = "json-stream";
+            // Create client config
+            PravegaConfig pravegaConfig =  PravegaConfig.fromDefaults()
+                        .withControllerURI(controllerURI)
+                        .withDefaultScope(scope)
+                        .withHostnameValidation(false);
```

Comment:
Why is this set? This makes things less secure. Our samples should not be insecure because people will be copying them.

Reply:
Removed  .withHostnameValidation(false);
PravegaConfig object created to use in 
Pravega pravega = new Pravega();
            pravega.tableSourceReaderBuilder()
                     .forStream(stream)
                    .withPravegaConfig(pravegaConfig);
This API allows only PravegaConfig